src/Evolution/Systems/GrMhd/ValenciaDivClean/System.hpp

```diff
@@ -35,6 +35,138 @@ namespace grmhd {
 ///
 namespace ValenciaDivClean {
 
+/*!
+ * \brief General relativistic magnetohydrodynamics (GRMHD) system with
+ * divergence cleaning coupled with electron fraction.
+ *
+ * We adopt the standard 3+1 form of metric
+ * \f{align*}
+ *  ds^2 = -\alpha^2 dt^2 + \gamma_{ij}(dx^i + \beta^i dt)(dx^j + \beta^j dt)
+ * \f}
+ * where \f$\alpha\f$ is the lapse, \f$\beta^i\f$ is the shift vector, and
+ * \f$\gamma_{ij}\f$ is the spatial metric.
+ *
+ * Primitive variables of the GRMHD system are
+ *  - rest mass density \f$\rho\f$
+ *  - electron fraction \f$Y_e\f$
+ *  - the spatial velocity \f{align*} v^i =
+ * \frac{1}{\alpha}\left(\frac{u^i}{u^0} + \beta^i\right) \f} measured by the
+ * Eulerian observer
+ *  - specific internal energy density \f$\epsilon\f$
+ *  - magnetic field \f$B^i = -^*F^{ia}n_a = -\alpha (^*F^{0i})\f$
+ *  - the divergence cleaning field \f$\Phi\f$
+ *
+ * with corresponding derived physical quantities which frequently appear in
+ * equations:
+ * - The transport velocity \f$v^i_{tr} = \alpha v^i - \beta^i\f$
+ * - The Lorentz factor \f{align*} W = -u^an_a = \alpha u^0 =
+ * \frac{1}{\sqrt{1-\gamma_{ij}v^iv^j}} = \sqrt{1+\gamma^{ij}u_iu_j} =
+ * \sqrt{1+\gamma^{ij}W^2v_iv_j} \f}
+ * - The specific enthalpy \f$h = 1 + \epsilon + p/\rho\f$ where \f$p\f$ is the
+ * pressure specified by a particular equation of state (EoS)
+ * - The comoving magnetic field
+ * \f{align*}
+ *  b^0 & = W B^i v_i / \alpha \\
+ *  b^i & = (B^i + \alpha b^0 u^i) / W \\
+ *  b^2 & = b^ab_a = B^2/W^2 + (B^iv_i)^2
+ * \f}
+ * - Augmented enthalpy density \f$(\rho h)^* = \rho h + b^2\f$ and augmented
+ * pressure \f$p^* = p + b^2/2\f$ which include contributions from magnetic
+ * field
+ *
+ * \note We are using the electromagnetic variables with the scaling convention
+ * that the factor \f$4\pi\f$ does not appear in Maxwell equations and the
+ * stress-energy tensor of the EM fields (geometrized Heaviside-Lorentz units).
+ * To recover the physical value of magnetic field in the usual Gaussian CGS
+ * unit, the conversion factor is
+ * \f{align*}
+ *  \sqrt{4\pi}\frac{c^4}{G^{3/2}M_\odot} \approx 8.35 \times 10^{19}
+ *  \,\text{Gauss}
+ * \f}
```

Comment:
Maybe add `That is, sqrt{b^2}_Gauss = sqrt{b^2}_code` or something like that to be explicit about how to convert?